templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+	<h1>Login</h1>
+	{% if form.errors %}
+		<p style="color: red;">Invalid credentials</p>
+	{% endif %}
+	<form action="{% url 'login' %}" method="post">
+		{% csrf_token %}
+		{{ form.as_p }}
+		<input type="submit" value="Submit">
+		<input type="hidden" name="next" value="{{ next }}">
```

Comment:
Ensure that the `next` variable is properly set in the context when rendering this template. This variable is used to redirect the user to the intended page after a successful login.